smart_contracts/contract_as/package.json

```diff
@@ -2,6 +2,12 @@
   "name": "casper-contract",
   "version": "1.0.0",
   "description": "Library for developing Casper smart contracts.",
+  "homepage": "https://docs.casperlabs.io/en/latest/dapp-dev-guide/writing-contracts/assemblyscript.html",
```

Comment:
How about we make the homepage https://docs.casperlabs.io/en/latest/dapp-dev-guide/index.html - let them know what they're in for.

Reply:
Sorry for the delay. This has been on my list. Sure. I will make the suggested change tomorrow.

Reply:
On second thoughts, I think the page  `https://docs.casperlabs.io/en/latest/dapp-dev-guide/writing-contracts/assemblyscript.html` seems to be more appropriate - because this is the npm package client - and the page you mention (`https://docs.casperlabs.io/en/latest/dapp-dev-guide/index.html`) is more general documentation.

Reply:
@dwerner 
Made the requested changes. Let's either close or merge this today. It's ope 24 days.